prestashopv1/plugin/modules/indodana/controllers/front/cancel.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+// phpcs:disable PSR1.Classes.ClassDeclaration.MissingNamespace
```

Comment:
Sorry, I'm using vscode's phpcs extension with PSR-2 standard, without this line, it will output error "Each class must be in a namespace of at least one level (a top-level vendor name)" in vscode.
But i think it's ok remove this line

Reply:
Hmm I think it's ok to keep this line.
Just add a code comment, for why to keep this line